### PR TITLE
Added billing route w/ billing iframe

### PR DESCRIPTION
**Pull request by allouis**
no-issue

### CODE REVIEW

File: app/components/gh-billing-iframe.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        let iframe = this.$('iframe')[0];
+        window.addEventListener('message', (event) => {
+            if (event && event.data && event.data.request === 'token') {
+                const ghostIdentityUrl = this.get('ghostPaths.url').api('ghost-identity');
+
+                this.ajax.post(ghostIdentityUrl).then(({token}) => {
+                    iframe.contentWindow.postMessage({
+                        request: 'token',
+                        repsonse: token
+                    }, '*');
```
Comment (allouis) [inline app/components/gh-billing-iframe.js]: This should not be `'*'` but the actual origin of the expected iframe

File: app/controllers/billing.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import Controller from '@ember/controller';
+import {alias} from '@ember/object/computed';
+
+export default Controller.extend({
+    guid: alias('model')
```
Comment (allouis) [inline app/controllers/billing.js]: I had an eslint error saying I must "alias" my model - so I copied this from controllers/site.js

File: app/routes/billing.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import AuthenticatedRoute from 'ghost-admin/routes/authenticated';
+
+export default AuthenticatedRoute.extend({
+    model() {
+        return (new Date()).valueOf();
```
Comment (allouis) [inline app/routes/billing.js]: :shrimp: Copied from site route

File: app/templates/components/gh-nav-menu.hbs [hunk 4]
```diff
@@ -83,6 +83,11 @@
                     </button>
                     <LinkTo @route="settings.labs" data-test-nav="labs">{{svg-jar "labs"}}Labs</LinkTo>
                 </li>
+                <li class="relative">
+                    <LinkTo @route="billing" data-test-nav="billing">
+                        {{svg-jar "house"}} View billing
```
Comment (allouis) [inline app/templates/components/gh-nav-menu.hbs]: Needs a different icon

File: app/components/gh-billing-iframe.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    didRender() {
+        let iframe = this.$('iframe')[0];
+        window.addEventListener('message', (event) => {
+            if (event && event.data && event.data.request === 'token') {
+                const ghostIdentityUrl = this.get('ghostPaths.url').api('ghost-identity');
+
+                this.ajax.post(ghostIdentityUrl).then(({token}) => {
+                    iframe.contentWindow.postMessage({
+                        request: 'token',
+                        response: token
```
Comment (allouis) [inline app/components/gh-billing-iframe.js]: Probably we want to actually design the protocol for communicating between the window->iframe

File: app/controllers/billing.js [hunk 6]
```diff
@@ -0,0 +1,6 @@
+import Controller from '@ember/controller';
```
Comment (allouis) [inline app/controllers/billing.js]: Is it possible to remove this entire file?

File: app/templates/components/gh-billing-iframe.hbs [hunk 7]
```diff
@@ -0,0 +1,15 @@
+<iframe id="billing-frame" class="billing-frame" src="http://localhost:5432" frameborder="0" allowtransparency="true"></iframe>
```
Comment (allouis) [inline app/templates/components/gh-billing-iframe.hbs]: `src` needs to be read from config

File: app/controllers/billing.js (hunk 6 above)
Comment (rishabhgrg) [reply app/controllers/billing.js]: @allouis Don't think so, as we'll need atleast the basic skeleton controller here with `guid` I think😬

**Comment by rishabhgrg** [comment]
@allouis Think this is good to merge on master in this state as this puts both billing UI and route behind config from server along with basic cleanup, let me know if you think anything else is needed :)

**Pull request merged**